### PR TITLE
[RC1][Ch01]: Fixing Typos

### PR DESCRIPTION
**Pull request by prabhubalan**
Fixing Typos and Sequencing the figure

### CODE REVIEW

File: doc/ref_cert/lfn/chapters/chapter01.md [hunk 1]
```diff
@@ -135,12 +135,12 @@ Perform VNF interoperability verifications against an implementation of CNTT ref
 
 In addition, respective Entrance criteria is a prerequisite which needs to be satisfied for NFVI+VNF to be considered **Testable**.
 
-**Dependencies** NFVI+VNF verification will rely upon test harnesses, test tools, and test suites provided by upstream OPNFV projects, including dovetaill, yardstick, and Bottleneck. These upstream projects will be reviewed semi-annually to verify they are still healthy and active projects. Over time, the projects representing the conformance process may change, but test parity is required if new test suites are added in place of older, stale projects.
+**Dependencies** NFVI+VNF verification will rely upon test harnesses, test tools, and test suites provided by upstream OPNFV projects, including dovetail, yardstick, and Bottleneck. These upstream projects will be reviewed semi-annually to verify they are still healthy and active projects. Over time, the projects representing the conformance process may change, but test parity is required if new test suites are added in place of older, stale projects.
```
Comment (lylavoie) [inline doc/ref_cert/lfn/chapters/chapter01.md:138]: Need to add Functest and xTesting into this list.  Dovetail should probably be listed as dovetail portal.

Comment (prabhubalan) [reply doc/ref_cert/lfn/chapters/chapter01.md:138]: Thanks @lylavoie. Please raise an issue to discuss further as this PR is to fix simple typos

Comment (prabhubalan) [reply doc/ref_cert/lfn/chapters/chapter01.md:138]: > Need to add Functest and xTesting into this list. Dovetail should probably be listed as dovetail portal.

Thanks @lylavoie. Please raise an issue to discuss further as this PR is to fix simple typos

Comment (prabhubalan) [reply doc/ref_cert/lfn/chapters/chapter01.md:138]: > Need to add Functest and xTesting into this list. Dovetail should probably be listed as dovetail portal.

Thanks @lylavoie. Please raise an issue to discuss further as this PR is to fix simple typos

**Pull request merged**